CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 
+# 0.19.0
+
+* `Export-Migration` will now allow references to objects in databases that have been applied `before` it.

```

Comment:
Why is before surrounded in backticks? That will format as code in a monospace font.